package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "app",
-  "version": "1.0.0",
+  "version": "1.2.3",
```

Comment:
I would remove `name` and `version` since we're not publishing a package and only using `package.json` for building stuff (cf. https://docs.npmjs.com/cli/v10/configuring-npm/package-json#version:~:text=to%20the%20version.-,If%20you%20don%27t%20plan%20to%20publish%20your%20package%2C%20the%20name%20and%20version%20fields%20are%20optional.,-Version%20must%20be).